go/service/main.go

```diff
@@ -227,6 +233,18 @@ func (d *Service) checkTrackingEveryHour() {
 	}()
 }
 
+func (d *Service) gregordConnect() error {
+	h := newGregorHandler(d.G())
+	d.G().Log.Debug("gregor URI: %s", d.G().Env.GetGregorURI())
+	rootCert, err := ioutil.ReadFile(os.Getenv("GREGOR_ROOT_CERT"))
+	if err != nil {
+		return err
+	}
+	// 9 parameters???
+	d.gregorConn = rpc.NewTLSConnection(d.G().Env.GetGregorURI(), rootCert, nil, h, true, nil, nil, d.G().Log, nil)
```

Comment:
As long as testing is still easy to fire off for everyone, and/or we automate it. Consider the case that you're explaining to a new employee of how to start up a local copy of keybase, with everything running reasonably.